legendary/cli.py

```diff
@@ -2625,6 +2626,58 @@ def move(self, args):
         self.core.install_game(igame)
         logger.info('Finished.')
 
+    def eula(self, args):
+        if not self.core.login():
+            logger.error('Login failed! Unable to check for EULAs.')
+            exit(1)
+        app_name = self._resolve_aliases(args.app_name)
+        game = self.core.get_game(app_name, update_meta=True)
+        if not game:
+            self.logger.error(f'No game found for "{app_name}"')
+            return
+        eulas = game.metadata.get('eulaIds') or ['$']
+
+        pattern = r'\w+'
+        keys = []
+        for eula in eulas:
+            keys += re.findall(pattern, eula)
+
+        not_accepted_eulas = []
+        for key in keys:
+            if args.epic and key == 'egstore':
+                continue
+            self.logger.debug(f'Fetching eula status for "{key}"')
+            eula = self.core.egs.eula_get_status(key)
+            if eula:
+                not_accepted_eulas.append(eula)
+
+        accepted = False
+
+        if not args.json:
+            for eula in not_accepted_eulas:
+                key = eula.get('key')
+                url = eula.get('url')
+                print(f' * EULA "{key}" - {url}')
```

Comment:
I've not looked at the shape of `eula` yet, but perhaps we could display something more descriptive than just the `key` here?

Reply:
I've included the EULA title here, it varries from generic `End User License Agreement` to `{GAME_TITLE} User Agreement` etc..